app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java

```diff
@@ -29,11 +29,17 @@
  * Created by manan on 22-05-2015.
  */
 public class BookmarksFragment extends Fragment {
-    SessionsListAdapter sessionsListAdapter;
+    static SessionsListAdapter sessionsListAdapter;
```

Comment:
I am referencing `notifyBookmarkedChanged()` from static context of  `onOptionsItemSelected(MenuItem item)` ..so need to keep `sessionsListAdapter` static.